README.md

```diff
@@ -4,27 +4,47 @@
 [![Coverage Status](https://coveralls.io/repos/github/zalando-incubator/postgres-operator/badge.svg)](https://coveralls.io/github/zalando-incubator/postgres-operator)
 [![Go Report Card](https://goreportcard.com/badge/github.com/zalando-incubator/postgres-operator)](https://goreportcard.com/report/github.com/zalando-incubator/postgres-operator)
 
-The Postgres operator manages PostgreSQL clusters on Kubernetes using the [operator pattern](https://coreos.com/blog/introducing-operators.html).
-During the initial run it registers the [Custom Resource Definition (CRD)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/#customresourcedefinitions) for Postgres.
-The `postgresql` CRD is essentially the schema that describes the contents of the manifests for deploying individual
-Postgres clusters using StatefulSets and [Patroni](https://github.com/zalando/patroni).
-
-Once the operator is running, it performs the following actions:
-
-* watches for new `postgresql` manifests and deploys new clusters
-* watches for updates to existing manifests and changes corresponding properties of the running clusters
-* watches for deletes of the existing manifests and deletes corresponding clusters
-* acts on an update to the operator configuration itself and changes the running clusters when necessary
-  (i.e. the Docker image changes for a minor release update)
-* periodically checks running clusters against the manifests and syncs changes
-
-Example: When a user creates a new custom object of type ``postgresql`` by submitting a new manifest with
-``kubectl``, the operator fetches that object and creates the required Kubernetes entities to spawn a new Postgres cluster
-(StatefulSets, Services, Secrets).
-
-Update example: After changing the Docker image inside the operator's configuration, the operator first goes to all StatefulSets
-it manages and updates them with the new Docker image; afterwards, all pods from each StatefulSet are killed one by one
-and the replacements are spawned automatically by each StatefulSet with the new Docker image. This is called the Rolling update.
+## Introduction 
+
+The Postgres [operator](https://coreos.com/blog/introducing-operators.html) manages PostgreSQL clusters on Kubernetes:
+
+1. The operator watches additions, updates, and deletions of PostgreSQL cluster manifests (descriptions) and changes the running clusters accordingly. For example, when a user submits a new manifest, the operator fetches that object and spawns a new Postgres cluster along with all necessary entities such as Kubernetes StatefulSets and Postgres roles. See this [Postgres cluster manifest](manifests/complete-postgres-manifest.yaml) for settings that a manifest may contain.
```

Comment:
s/that object/that manifest

Reply:
"(descriptions)" I would also scratch that, manifest is such a common word.

Reply:
But please split this into multiple lines in the file. once sentence per line.